docs/core/diagnostics/built-in-metrics-aspnetcore.md

```diff
@@ -1,41 +1,81 @@
 ---
-title: ASP.NET Core Metrics
+title: ASP.NET Core metrics
 description: Review the metrics available for ASP.NET Core
 ms.topic: reference
-ms.date: 10/16/2023
+ms.date: 11/02/2023
 ---
 
-# ASP.NET Core Metrics
+# ASP.NET Core metrics
 
 This article describes the metrics built-in for ASP.NET Core produced using the
 <xref:System.Diagnostics.Metrics?displayProperty=nameWithType> API. For a listing of metrics based on the older [EventCounters](event-counters.md) API,
 see [here](available-counters.md).
 
+- [Meter: `Microsoft.AspNetCore.HeaderParsing`](#meter-microsoftaspnetcoreheaderparsing)
```

Comment:
Since `Microsoft.AspNetCore.HeaderParsing` is a part of dotnet/extensions, shouldn't we move description of this meter and its metrics to ".NET extensions metrics" article? @IEvangelist

Reply:
That's a fair question, but I'd argue "no". This package, while it ships out of dotnet/extensions is specific to ASP.NET Core and isn't workload agnostic, therefore it belongs with the other ASP.NET Core specific metrics (otherwise we're shipping our org chart so to speak). I suppose we should defer to @davidfowl and @joperezr, but I feel very strongly about this.